hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -153,6 +154,10 @@
   private static final String MERGE_ALLOW_DUPLICATE_ON_INSERTS = "hoodie.merge.allow.duplicate.on.inserts";
   private static final String DEFAULT_MERGE_ALLOW_DUPLICATE_ON_INSERTS = "false";
 
+  // Enable usage of RocksDb for External Spillable Map
+  public static final String DEFAULT_SPILLABLE_DISK_MAP_TYPE = ExternalSpillableMap.DiskMapType.ROCK_DB.name();
```

Comment:
The default will be changes from ROCKS_DB to DISK_MAP. Its intentionally set to ROCKS_DB to ensure all tests pass.

Reply:
lets do the flip now, that all the tests pass with rocksdb map?